Patch orginially appeared in fc7:

* Wed Jan 25 2006 Jeremy Katz <katzj@redhat.com> - 0.16.17-2
- build on x86_64
- don't build elks (it's not happy on x86_64)

--- dev86-0.16.17/makefile.in
+++ dev86-0.16.17/makefile.in
@@ -89,10 +89,10 @@
 
 #ifdef GNUMAKE
 all: check_config bcc86 cpp unproto copt as86 ar86 ld86 objdump86 \
-     library lib-bsd alt-libs elksemu
+     library lib-bsd alt-libs 
 
 install: check_config install-bcc install-man \
-     install-lib install-emu
+     install-lib
 
 install-all: install install-other
 
